src/helpers/safari-browser-configurator.helper.js

```diff
@@ -0,0 +1,6 @@
+import shell from 'shelljs';
+import path from "path";
+
+export const safariBrowserConfigurator = (config) => {
+  shell.exec(`defaults write -app Safari DownloadsPath ${path.join(config.projectPath, config.downloads)}`);
```

Comment:
This has to be done through command line? There is no option to pass it with options?

Reply:
Unfortunately, I did not see the possibility to configure it in the same way as for Firefox or Google Chrome